projects/1.19/assets/magnum-torch-forge/magnumtorch/lang/zh_cn.json

```
@@ -0,0 +1,15 @@
+{
+  "block.magnumtorch.diamond_magnum_torch": "钻石卓越火炬",
+  "block.magnumtorch.emerald_magnum_torch": "绿宝石卓越火炬",
+  "block.magnumtorch.amethyst_magnum_torch": "紫水晶卓越火炬",
+  "block.magnumtorch.magnum_torch.info": "阻止生物大范围生成。",
```

```suggestion
  "block.magnumtorch.magnum_torch.info": "阻止生物在大范围内生成。",
```
嗯……只能说改前有种说不上来的怪异，理论上不改也行？

---

原文：**Prevents mob spawns in a huge area.**

```suggestion
  "block.magnumtorch.magnum_torch.info": "在一个很大的范围内阻止生物生成。",
```